ntp-daemon/src/main.rs

```diff
@@ -23,5 +27,66 @@ async fn main() -> Result<(), Box<dyn Error>> {
     // this away when not using sentry.
     let _guard = finish_tracing_init(&mut config, has_log_override)?;
 
-    ntp_daemon::spawn(&config.system, &config.peers).await
+    let peers_reader = Arc::new(tokio::sync::RwLock::new(Vec::new()));
+    let peers_writer = peers_reader.clone();
+
+    std::fs::remove_dir_all("/run/ntpd-rs")?;
+    std::fs::create_dir("/run/ntpd-rs")?;
```

Comment:
Removing and recreating the directory feels like work that shouldn't be the daemons responsibility. Also, if we do this I would strongly suggest we make the path configurable.

Reply:
Ok, having looked into the options around security some more, I am going to take the stronger position that we should leave creating the /run directory to systemd, as this makes running with minimal permissions a lot easier.

Reply:
and we panic when the path doesn't exist?

Reply:
That seems to me the only reasonable option.